packages/liquidator/test/Liquidator.js

```diff
@@ -1206,12 +1204,12 @@ contract("Liquidator.js", function(accounts) {
           // show position has been extended
           assert.equal(
             sponsor2Positions.withdrawalRequestPassTimestamp.toNumber(),
-            Number(sponsor2Liquidations[0].liquidationTime) + Number(liquidationLiveness)
+            Number(sponsor2Liquidations[1].liquidationTime) + Number(withdrawLiveness)
```

Comment:
This was the false positive test: we were checking the withdrawalRequestPassTimestamp of the wrong liquidation struct.

At this point in the test, the sponsor's `withdrawalRequestPassTimestamp`  has been reset to the latest liquidation time (`sponsor2Liquidations[1].liquidationTime`) plus the `withdrawalLiveness`.

Previously, this test was "correct" because the liquidation was done pre-withdrawal-expiry, meaning that the `withdrawalRequestPassTimestamp` was not actually extended. Therefore, the sponsor's `withdrawalRequestPassTimestamp` should remain equal to the original liquidation time (`sponsor2Liquidations[0].liquidationTime`) + the original liveness.*

*Original liquidation done on line 1167